src/main/java/io/github/hapjava/characteristics/impl/base/BaseCharacteristic.java

```diff
@@ -113,7 +113,7 @@ public BaseCharacteristic(
                       .add("type", shortType)
                       .add("perms", perms.build())
                       .add("format", format)
-                      .add("ev", false)
+                      //                      .add("ev", false)
```

Comment:
Was it your intention to comment  this out? Looks like a rather common piece of code.